tests/Models/User.php

```diff
@@ -30,22 +30,24 @@
  * @property string $username
  * @property MemberStatus member_status
  */
-class User extends Eloquent implements AuthenticatableContract, CanResetPasswordContract
+class User extends Model implements AuthenticatableContract, CanResetPasswordContract
 {
+    use DocumentModel;
```

Comment:
All the test models have been updated to use the trait, so that we are sure all the features works without extending `MongoDB\Laravel\Eloquent\Model`.

Reply:
Having one test with a model that extends our document model class instead of using the trait will help us prevent regressions in our model class.